lib/rubocop/cop/rspec/mixin/skip_or_pending.rb

```diff
@@ -10,13 +10,33 @@ module SkipOrPending
         # @!method skipped_in_metadata?(node)
         def_node_matcher :skipped_in_metadata?, <<-PATTERN
           {
-            (send _ _ <#skip_or_pending? ...>)
-            (send _ _ ... (hash <(pair #skip_or_pending? { true str dstr }) ...>))
+            (send _ _ <#skip_or_pending_sym? ...>)
+            (send _ _ ... (hash <(pair #skip_or_pending_sym? { true str dstr }) ...>))
           }
         PATTERN
 
-        # @!method skip_or_pending?(node)
-        def_node_matcher :skip_or_pending?, '{(sym :skip) (sym :pending)}'
+        # @!method skip_or_pending_sym?(node)
+        def_node_matcher :skip_or_pending_sym?, '{(sym :skip) (sym :pending)}'
+
+        # @!method skip_or_pending_inside_block?(node)
+        #   Match examples defined inside a block which is not a hook
+        #
+        #   @example source that matches
+        #     context 'when color is blue' do
+        #       skip 'not implemented yet'
+        #     end
+        #
+        #   @example source that does not match
+        #     pending 'not implemented yet'
+        #     it 'does not match' do
+        #       skip { }
```

Comment:
Why do we ignore this? Even though we have `RSpec/SkipBlockInsideExample` cop, it may be turned off and this syntax won't cause RSpec to blow up.

Unrelated to this refactoring, an idea for a follow-up improvement.

Reply:
Surely that is a wrong. I updated it.